README.md

```diff
@@ -318,12 +325,28 @@ $ firefox keywords.html
 
 Or use the provided `create-docs.sh` script, which automatically concatenates
 all of the keyword-containing libraries from `lib/` directory with
-`keywords.robot`, and generates one big html file containing all the keywords
-within this repo:
+`keywords.robot`, and generates one big html file containing all the
+keywords within this repo.
+
+It also allows to choose the version of the
+documentation, which basically decides if you want to perform the UEFI
+configuration via "UEFI setup app" (`UEFI`) or via "Dasharo Config
+Utility tool" (`DCU`):
+
+```bash
+$ ./scripts/create-docs.sh UEFI
+Generating UEFI version of the documentation
+Documentation generated and saved as all-keywords.html
+$ firefox all-keywords.html
+```
+
+Or
 
 ```bash
-$ ./scripts/create-docs.sh
-Documentation generated and saved as docs/index.html
+$ ./scripts/create-docs.sh DCU
+Generating DCU version of the documentation
+Documentation generated and saved as all-keywords.html
+$ firefox all-keywords.html
```

Comment:
```suggestion
Generating DCU version of the documentation
[DEBUG] Skip uefi-setup-menu.robot
Documentation generated and saved as ./docs/index.html
$ firefox docs/index.html
```

Reply:
Changed here: https://github.com/Dasharo/open-source-firmware-validation/pull/596/commits/718a2f4bf1f766e056609cf1db8d977c364f8b5a
@EduKav1813 you should've mentioned it here